### PR TITLE
multisigma mode added

### PR DESCRIPTION
**Pull request by jedori0228**
`kMultisigma` added in `sbnanaobj`, followed by https://github.com/SBNSoftware/sbncode/pull/233.

### CODE REVIEW

File: sbnanaobj/StandardRecord/SREnums.h [hunk 1]
```diff
@@ -221,7 +221,7 @@ namespace caf
   };// g4_process_
 
   enum ReweightType_t{
-    kMultiSim, kPMNSigma, kFixed, kDefault
+    kMultisim, kPMNSigma, kMultisigma, kFixed, kDefault
```
[the remark this answers is not in the document]
Comment (jedori0228) [reply sbnanaobj/StandardRecord/SREnums.h:224]: Oh I should've not changed the Upper cases, this was done by a mistake! I will updated this. Also the only existing usage of `ReweightType_t` I see is at https://github.com/SBNSoftware/sbncode/blob/1d633011bbedac8b80ad3dbf6e37de21d32ab814/sbncode/CAFMaker/FillTrue.cxx#L74

[the remark this answers is not in the document]
Comment (jedori0228) [reply sbnanaobj/StandardRecord/SREnums.h:224]: Oh I should've not changed the Upper cases, this was done by a mistake! I will updated this. Also the only existing usage of `ReweightType_t` I see is at https://github.com/SBNSoftware/sbncode/blob/1d633011bbedac8b80ad3dbf6e37de21d32ab814/sbncode/CAFMaker/FillTrue.cxx#L74, which is a direct conversion from `EventWeightParameterSet:: rwtype` defined in `sbnobj` (https://github.com/SBNSoftware/sbnobj/blob/v09_13_04/sbnobj/Common/SBNEventWeight/EventWeightParameterSet.h#L59). So we have to match the order of `ReweightType_t` to `EventWeightParameterSet:: rwtype`, right?

[the remark this answers is not in the document]
Comment (cjbackhouse) [reply sbnanaobj/StandardRecord/SREnums.h:224]: Ah, yes. It's worth a comment at the top of `ReweightType_t` explaining that too.

**Review by mroda88 — APPROVED** [review]
LGTM

File: sbnanaobj/StandardRecord/SREnums.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (jedori0228) [reply sbnanaobj/StandardRecord/SREnums.h]: As we discussed in Slack, I moved `kDefault` to -1 and put `kMultiSigma` next to `kFixed`. Also add a comment that this should be synchronized to `sbnobj/Common/SBNEventWeight/EventWeightParameterSet.h`.

**Pull request closed without merging**